common/app/renderers/DotcomRenderingService.scala

```diff
@@ -133,6 +129,23 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     post(ws, json, Configuration.rendering.baseURL + "/Article", page)
   }
 
+  def getBlocks(
+                 ws: WSClient,
+                 page: LiveBlogPage,
+                 blocks: Seq[Block],
+                )(implicit request: RequestHeader): Future[String] = {
+
+    val dataModel = DotcomBlocksRenderingDataModel.forLiveblog(page, blocks, request)
+    val json = DotcomBlocksRenderingDataModel.toJson(dataModel)
+
+    ws
```

Comment:
The HTML returned from DCR needs to be wrapped in a JSON object which is why the ordinary `post` can't be used.